plugins.json

```diff
@@ -622,5 +622,17 @@
         "download": "https://github.com/funktechno/kanboard-plugin-wiki/releases/download/0.2.5-alpha/Wiki-0.2.5.zip",
         "remote_install": true,
         "compatible_version": ">=1.0.37"
+    },
+    "weightedvoting": {
+        "title": "Weighted voting",
+        "version": "1.0.0",
+        "author": "Manel Perez",
+        "license": "MIT",
+        "description": "This plugin add new features to Kanboard. Weighted voting for the evaluation of group activities in Kanboard.",
+        "homepage": "https://github.com/manelperez/kanboard-plugin-voting",
+        "readme": "https://github.com/manelperez/kanboard-plugin-voting/blob/master/README.md",
+        "download": "https://github.com/manelperez/kanboard-plugin-voting/archive/voting-v1.0.0.zip",
```

Comment:
You should make your own zip archive because the directory structure of GitHub archives is not the same as the one expected by Kanboard. If you don't want to make your own archive, set to false "remote_install". Take a look to the footer notes here https://github.com/kanboard/website#how-to-add-a-new-plugin-to-the-list